PowerShellTools.HostService/ServiceManagement/Debugging/PowershellDebuggingServiceUtilities.cs

```diff
@@ -345,5 +346,44 @@ private void AddEventHandlers()
             _runspace.StateChanged += _runspace_StateChanged;
             _runspace.AvailabilityChanged += _runspace_AvailabilityChanged;
         }
+
+        /// <summary>
+        /// Invokes given script on the provided PowerShell object after setting its runspace object.
+        /// </summary>
+        /// <param name="powerShell">This should usually be _currentPowerShell. Make sure to enclose uses of _currentPowerShell and this method
+        /// inside of a using. </param>
+        /// <param name="script">Script to invoke.</param>
+        /// <returns>Returns the result of the invoke</returns>
+        private Collection<PSObject> InvokeScript(PowerShell powerShell, string script)

```

Comment:
Yes:

1. Incase the user of said function wanted to do something with _currentPowerShell after invoking their script on it
2. Because calling this method multiple times would essentially create a bunch of using blocks one after another which seems inefficient/silly